.github/workflows/build.yml

```
@@ -82,6 +86,9 @@ jobs:
         uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
         with:
           go-version: ${{ matrix.go-version }}
+      - name: log exact golang version
```

Are you planning to keep the commit to print the go version? I'm fine either way. If we keep it, just a nit to add a new line between the steps
```suggestion

      - name: log exact golang version
```

---

@travisn  wrote:

> Are you planning to keep the commit to print the go version? I'm fine either way.

It was only added to troubleshoot this PR an does not seem to provide long-term value, so I removed it.